Program.cs

```diff
@@ -1,10 +1,12 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// To check whether the given input is a palindrome or not
+
+Console.Write ("Enter something to check whether it is a palindrome or not: ");

```

Comment:
Fix this exception.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/22f65e83-abb2-48fd-8856-a42bef55b1e2)